### ISSUE TITLE
Build test script for tab component

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/vets-website#7516]
### Criteria
We should be able to write test scripts for more complicated elements, starting with tabs. These will be more difficult, and will need to prescribe to several boundaries:
* Elements under test should have a custom `data-testAttr` attribute for our script to evaluate
* Elements should have a second `data-typeAttr` attribute for our script to know what test to run
* Elements should follow the WAI-ARIA recommendations for user interaction, or another documented set of interaction criteria, so test results can be repeated over many runs.
